lib/src/config/config_file/client.example.toml

```
@@ -39,12 +39,14 @@ seed_nodes = [
 
 ##############################################################################
 #
-# Identity file (PCKS#12) and password for private key
+# TLS network configuration:
+# - Identity file (PEM-encoded X.509 format)
+# - private key (PEM-encoded ASN.1 in either PKCS#8 or PKCS#1 format)
 #
 ##############################################################################
 #[network.tls]
-#identity_file = "./my.domain.p12"
-#identity_password = "secret"
+#private_key = "./my_private_key.pem"
+#certificate = "./my_certificate.pem"
```

The config setting is called `certificates`, plural:
```suggestion
#certificates = "./my_certificate.pem"
```